QUARKUS-6258.md

```diff
@@ -15,9 +15,16 @@ The list of prioritized features (taken from the issue):
 - Advanced RAG
 - Guardrails 	
 - WebSocket chat integration 	
-- CDI Chat Model.
+- CDI Chat Model
 
-We also need to verify correct usage of OpenAI and Watson.x (optional for the first release) APIs as well as observability (metrics, logging, and telemetry).
+We also need to verify correct usage of OpenAI and Watson.x (optional for the first release) APIs as well as observability (metrics, logging, and telemetry), the latter being a requirement for full support.
```

Comment:
This is really vaguely worded. Let's be precise in technical writing, please.

What are the scenarios and what is the correct usage in those scenarios? What are the integration points?

Reply:
> correct usage

The app can send queries to these APIs, receive answers and not fail or loose data in the process.